astroquery/mast/observations.py

```diff
@@ -1121,6 +1121,63 @@ def service_request_async(self, service, params, *, pagesize=None, page=None, **
 
         return self._portal_api_connection.service_request_async(service, params, pagesize, page, **kwargs)
 
+    def _normalize_filter_value(self, key: str, val) -> list:
+        """
+        Normalize a filter value into a list suitable for MAST filters.
+
+        Parameters
+        ----------
+        key : str
+            Parameter name (used for error messages).
+        val : any
+            Raw filter value.
+
+        Returns
+        -------
+        list
+            Normalized filter values.
+        """
+        # Range filters must be dicts with 'min' and 'max'
+        if isinstance(val, dict):
+            if not {"min", "max"}.issubset(val.keys()):
```

Comment:
What happens when value has more than min/max, would the server handle dropping/using the extra ones?